src/NServiceBus.Core/Recoverability/Faults/MoveFaultsToErrorQueueBehavior.cs

```diff
@@ -71,6 +90,7 @@ public MoveFaultsToErrorQueueBehavior(CriticalError criticalError, IPipelineBase
         HostInformation hostInformation;
         BusNotifications notifications;
         string errorQueueAddress;
+        readonly FaultsStatusStorage faultsStatusStorage;
```

Comment:
not that I'm aware of. It's more of an unwritten rule. Is it R# that always uses readonly or could we disable that @SimonCropp ?